src/main/java/com/gary/backendv2/model/dto/request/AddAmbulanceRequest.java

```diff
@@ -21,7 +21,7 @@ public class AddAmbulanceRequest {
     private Integer seats;
 
     @NotBlank
-    private String longitude;
+    private double longitude;
```

Comment:
```suggestion
    private Double longitude;
```
Double class pls później jest nullability sprawdzane tych pól, a typ double nie jest nullowalny